.travis.yml

```
@@ -4,7 +4,7 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
+  - 1
```

Travis takes this to mean the "the latest version with major version 1" right now 1.4.2. We should be able to support older versions.
```suggestion
  - 1.0
```

---

Maybe we should do 1 and 1.0 to test both the latest Julia version and the LTS 1.0

---

I agree, looks like other packages are running 1, 1.0 and nightly.

---

Upstream, this is what the Travis setup looks like:
https://github.com/JuliaInterop/JavaCall.jl/blob/master/.travis.yml

We test 1.0, 1, nightly across Linux, Windows, and Mac, and also test JDK 8 and 11